op-service/crypto/espresso.go

```diff
@@ -55,8 +56,10 @@ func (c *clientSigner) SignTransaction(ctx context.Context, address common.Addre
 
 var _ ChainSigner = &clientSigner{}
 
-// PrivateKeySignerFn creates a SignerFn that signs transactions with the given
-// private key.
+// privateKeySigner creates a is a ChainSigner that delegates to the stored
```

Comment:
Nit: Remove "creates a".

Reply:
Fixed in a817ba4e53dadfd4589b1763272fcfba40941369